### PR TITLE
Adding cards for ttbar Dark Matter sample

### PR DESCRIPTION
**Pull request by sandeepbhowmik1**

### CODE REVIEW

**Comment by menglu21** [comment]
Hi @sandeepbhowmik1 please use template cards together with script, we don't want to review 300 cards

**Comment by sandeepbhowmik1** [comment]
Hi @menglu21 Can you please have a look at it 
Thanks

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_dilepton_pseudoscalar/templatecards/ttbarDM__dilepton__DMsimp_LO_ps_spin0_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 1 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 5 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ True = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 20.0   = xqcut   ! minimum kt jet measure between partons
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_dilepton_pseudoscalar/templatecards/ttbarDM__dilepton__DMsimp_LO_ps_spin0_run_card.dat:78]: is "20" the optimized value by checking the DJR plot

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_dilepton_scalar/templatecards/ttbarDM__dilepton__DMsimp_LO_s_spin0_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 1 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 5 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ True = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 20.0   = xqcut   ! minimum kt jet measure between partons
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_dilepton_scalar/templatecards/ttbarDM__dilepton__DMsimp_LO_s_spin0_run_card.dat:78]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_inclusive_pseudoscalar/templatecards/ttbarDM__inclusive__DMsimp_LO_ps_spin0_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 1 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 5 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ True = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 20.0   = xqcut   ! minimum kt jet measure between partons
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_inclusive_pseudoscalar/templatecards/ttbarDM__inclusive__DMsimp_LO_ps_spin0_run_card.dat:78]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_inclusive_scalar/templatecards/ttbarDM__inclusive__DMsimp_LO_s_spin0_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 1 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 5 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ True = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 20.0   = xqcut   ! minimum kt jet measure between partons
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttbarDM/ttbarDM_inclusive_scalar/templatecards/ttbarDM__inclusive__DMsimp_LO_s_spin0_run_card.dat:78]: same question here

**Comment by sandeepbhowmik1** [comment]
Hi @menglu21 

I have taken the code used in Run2.

So I tried to contact people used these code in Run2. 
There were some study but we couldn't find those old plots.

Thanks

**Pull request merged**